Detectors/TRD/base/include/TRDBase/Tracklet.h

```diff
@@ -101,8 +109,8 @@ class Tracklet
   int getNHits0() const { return mNHits0; }
   int getNHits1() const { return mNHits1; }
 
-  unsigned int getTrackletWord() const { return mTrackletWord; }
-  void setTrackletWord(unsigned int trackletWord) { mTrackletWord = trackletWord; }
+  unsigned int getTrackletWord() const;
+  void setTrackletWord(unsigned int trackletWord) { LOG(fatal) << "This is no longer permitted, o2::trd::Tracklet::setTrackletWord"; }
```

Comment:
why not remove this function. then nobody can call it.

Reply:
cool will remove.

Reply:
removed